### ISSUE TITLE
[Story] See a list of active institutions I need to file for this filing season

### ISSUE THREAD

**Issue by angelcardoz** [cfpb/sbl-project#120]
### Epic

Epic issue includes overview, complete list of user stories, technical and content requirements, user testing task ideation, and current design mock-ups.

[Epic] Small business lending filing home https://github.com/cfpb/sbl-project/issues/14

### User story

As a filer, I want to see a list of active institutions I need to file for this filing season, so that I can select the appropriate financial institution.

### Acceptance criteria

Given [some context or conditions], when [the action that is performed], then [the expected observable outcome or behavior].

### Technical tasks

Developers should create technical tasks and add links (below) prior to sprint planning.

#### Front end
- [ ] Task (link)
- [ ] Task (link)

#### Back end
- [ ] Task (link)
- [ ] Task (link)

**Comment by natalia-fitzgerald** [comment]
@angelcardoz 
My interpretation of this user story is that a user should be able to select a filing period (year) and see a list of the financial institutions that are associated with their user profile. This list will represent the financial institutions they need to file for. My understanding is that for MVP we will default to the current filing period which would be data collected in 2023 that must be filed in 2024. Because of this we will not include the date select component for MVP (because only 2023 data will be available). Is this accurate?

**Comment by angelcardoz** [comment]
@natalia-fitzgerald  That's correct, for MVP we will default to the current filing period which would be data collected in 2024* that must be filed in 2025*. Because of this we will not include the date select component for MVP (because only 2025* data will be available). Also, I do see some issues with the story, it should not convey "date select component". Here is the updated story and modified acceptance criteria. Please let me know your thoughts:

Story: "As a filer, I want to see all associated financial institutions I need to file for this filing season, so that I know which financial institutions I need to file for based on information available to CFPB"
  
Acceptance criteria: "Given a system user is logged in, when they visit the small business lending filing home page, then the system displays all active associated financial institutions they need to file for this filing season,  each in their respective card component,  as well as each financial institution's respective LEI."

*updated dates based on latest discussion with Monica, Hans, and Alex.
